ContinuousRegistration/Source/make_registration_scripts.py

```diff
@@ -92,15 +86,15 @@ def run(parameters):
                     output_directory = os.path.join(parameters.output_directory,
                                                     team_name, blueprint_name)
 
-                    if parameters.make_shell_scripts:
+                    if parameters.make_scripts == 'shell':
                         dataset.make_shell_scripts(parameters.superelastix,
                                                    blueprint_file_name, file_names,
-                                                   output_directory)
+                                                   output_directory, '.sh')
 
-                    if parameters.make_batch_scripts:
-                        dataset.make_batch_scripts(parameters.superelastix,
+                    if parameters.make_scripts == 'batch':
```

Comment:
`elif` makes it easier to read or rather something like
```
if parameters.make_scripts in ('shell', 'batch'):
    ext = '.sh' if parameters.make_scripts == 'shell' else '.bat'
    dataset.make_shell_scripts(parameters.superelastix, blueprint_file_name, file_names, output_directory, ext)
```

Reply:
Thanks for the suggestion!